server/routers/alerts.py

```
@@ -94,8 +96,12 @@ def get_unread_count(
             raise HTTPException(status_code=404, detail="User not found")
 
         count = (
-            db.query(AlertQueue)
-            .filter(AlertQueue.user_id == user_id, AlertQueue.status == "pending")
+            db.query(Alert)
+            .join(AlertQueue, Alert.id == AlertQueue.alert_id)
+            .filter(
+                AlertQueue.user_id == user_id,
+                Alert.is_read == False
```

Ruff (0.14.1)

Avoid equality comparisons to `False`; use `not Alert.is_read:` for false checks

Replace with `not Alert.is_read`

(E712)